src/ray/test/run_core_worker_tests.sh

```diff
@@ -22,7 +22,7 @@ fi
 set -e
 set -x
 
-bazel build -c dbg $RAY_BAZEL_CONFIG "//:core_worker_test" "//:mock_worker"  "//:raylet" "//:raylet_monitor" "//:libray_redis_module.so" "@plasma//:plasma_store_server"
+bazel build -c dbg $RAY_BAZEL_CONFIG "//:core_worker_test" "//:mock_worker"  "//:raylet" "//:raylet_monitor" "//:gcs_server" "//:libray_redis_module.so" "@plasma//:plasma_store_server"
```

Comment:
why is this needed?

Reply:
if we enable gcs service, we need start gcs server before start raylet.